modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/40_dhcp.js

```diff
@@ -43,25 +96,37 @@ return baseclass.extend({
 			else
 				exp = '%t'.format(lease.expires);
 
-			return [
+			rows = [
 				lease.hostname || '-',
 				lease.ipaddr,
 				lease.macaddr,
 				exp
 			];
-		}), E('em', _('There are no active leases')));
+
+			if (lease.macaddr != null) {
+				var mac = lease.macaddr.toUpperCase();
+				rows.push(E('button', {
+					'class': 'cbi-button cbi-button-apply',
+					'click': L.bind(this.handleCreateStaticLease, this, lease),
```

Comment:
Please use `ui.createHandlerFn(this, 'handleCreateStaticLease', lease)` here (see [`ui.createHandlerFn()`](https://openwrt.github.io/luci/jsapi/LuCI.ui.html#createHandlerFn))

Reply:
I use `bind` instead of create `createHandler` since with createHandler the 'set Static' button is enabled again after the function is completed. Looks bad that you click the Button. Then it became enabled and when the pool function refresh the data the button became disabled.

Reply:
I see, it is fine then.